host/xtest/symm_cipher_perf.c

```diff
@@ -554,7 +613,9 @@ int aes_perf_runner_cmd_parser(int argc, char *argv[])
 	int verbosity = CRYPTO_DEF_VERBOSITY;	/* Verbosity (-v) */
 	int decrypt = 0;		/* Encrypt by default, -d to decrypt */
 	int keysize = AES_128;	/* AES key size (-k) */
-	int mode = TA_AES_ECB;	/* AES mode (-m) */
+	int cipher = AES;
+	int mode = ECB;
+	int algo = TA_AES_ECB;	/* AES algo */
```

Comment:
I suggest to initialize `alog` to -1. It's value will always be set based on `cipher` and `mode` value.

Reply:
solved